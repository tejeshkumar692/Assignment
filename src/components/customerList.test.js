import React from 'react'
import { render } from '@testing-library/react';
import CustomerList from './customerList';

describe('<CustomerList /> spec', () => {
    it('renders the component', () => {
        const container = render(<CustomerList />)
        expect(container.firstChild).toMatchSnapshot()
    })
});
