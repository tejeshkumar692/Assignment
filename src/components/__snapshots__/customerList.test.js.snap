// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CustomerList /> spec renders the component 1`] = `undefined`;
